cmd/internal/cli/plugin_enable_linux.go

```diff
@@ -0,0 +1,34 @@
+package cli
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/sylabs/singularity/docs"
+	"github.com/sylabs/singularity/internal/app/singularity"
+	"github.com/sylabs/singularity/internal/pkg/buildcfg"
+	"github.com/sylabs/singularity/internal/pkg/sylog"
+)
+
+// PluginEnableCmd enables the named plugin
+//
+// singularity plugin enable <name>
+var PluginEnableCmd = &cobra.Command{
+	Run: func(cmd *cobra.Command, args []string) {
+		err := singularity.EnablePlugin(args[0], buildcfg.LIBEXECDIR)
+		if err != nil {
+			if os.IsNotExist(err) {
+				sylog.Fatalf("Failed to enable plugin %q: plugin not found.", args[0])
+			} else {
```

Comment:
`} else {` not necessary here. Might be simpler as:

```
if os.IsNotExist(err) {
    sylog.Fatalf([...])
}
sylog.Fatalf([...])
```

Reply:
I was trying to make it non-ambiguous that it's handling "all the other errors" without forcing the reader to keep in mind that "sylog.Fatalf" interrupts the program, but I can change this.

Reply:
Changed it to what @bauerm97 is asking for.